tests/test_data/test_filter.py

```diff
@@ -1,7 +1,10 @@
+import unittest
+
+import pandas as pd
 import polars as pl
 import pytest
 
-from ..context import nhs
+import nhs.data.filter
```

Comment:
Does this actually work? You can't import from `nhs` normally which is why we needed to import from `context`